### PR TITLE
Visual feedback after creating, editing and deleting a service or entity

### PR DESCRIPTION
**Pull request by Shaky212**
see: https://www.pivotaltracker.com/story/show/182246029

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ entity.delete.request.ticket.description: "h2. Details
 *Applicant name*: %applicant_name%
 *Applicant email*: %applicant_email%.
 *Entity name (en)*: %entity_name%."
-
+entity.delete.flash.message: "Your entity is deleted."
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml:19]: Perhaps, for the sake of consistency: rename the translation key to read: `entity.delete.flash.success`.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/FlashMessage/flashMessages.html.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% set flashes = app.session.flashbag.all %}
```
Comment (MKodde) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/FlashMessage/flashMessages.html.twig:1]: Thanks for extracting the flash message into a partial of its own! I see you removed the `card messages` class from the container `div` element. Was that intentional? As they seem to have changed the look and feel of the messages. Either use the old look and feel. Or clean up the now unused classes from the SCSS stylesheet.

**Review by MKodde — COMMENTED** [review]
Great work once again! See some feedback below. 

Additionally, I wonder if you tried to add some web tests coverage on the new flash messages you added. If you did not, maybe it's easy to do so.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml (hunk 1 above)
Comment (Shaky212) [reply src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml:19]: Agree.

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/FlashMessage/flashMessages.html.twig (hunk 2 above)
Comment (Shaky212) [reply src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/FlashMessage/flashMessages.html.twig:1]: On closer inspection, the card messages aren't that bad after all and line up nicely with the rest of the page. So revert to the old look and feel.

**Pull request merged**